src/App.tsx

```diff
@@ -1,14 +1,54 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import {getTodos} from './api'
+import { Todo } from './types/Todo';
+import { SortType } from './enums';
+
+
+
 
 export const App: React.FC = () => {
+  const [todosFromServer, setTodoFromServer] = useState<Todo[]>([]);
+  const [sortType, setSortType] =useState(SortType.All);
+  const [query, setQuery] =useState('');
+  const [isShow, setIsShow] = useState(false);
+  const [selectedUserId, setSelectedUserId] = useState(0);
+
+  const selectedTodo = todosFromServer.find(todo => todo.id === selectedUserId) || null;
```

Comment:
why did you use selectedUserId variable to find selected todo? probably selected todo id should be here?